amqp_influxdb/__init__.py

```diff
@@ -44,6 +45,24 @@ def __init__(self,
 
         connection_parameters = connection_parameters or {}
 
+        if isinstance(connection_parameters.get('credentials'), dict):
```

Comment:
The user should be able to supply a user and password for Rabbit in the manager blueprint's inputs file (and those inputs default to a preset rabbitmq user and pass).